Ignore font change requests that are note followed by any words (closes: #489181).

diff -ruN -x '*.rej' -x '*.orig' man-1.6f-old/man2html/man2html.c man-1.6f/man2html/man2html.c
--- man-1.6f-old/man2html/man2html.c	2008-09-07 10:56:00.000000000 +0200
+++ man-1.6f/man2html/man2html.c	2008-09-07 10:56:54.000000000 +0200
@@ -2131,7 +2131,10 @@
 		char font[2];
 		font[0] = c[0]; font[1] = c[1];
 		c = c+j;
-		if (*c == '\n') c++;
+		if (*c == '\n') {
+		    c++;
+		    break;
+		}
 		sl = fill_words(c, wordlist, SIZE(wordlist), &words, '\n');
 		c = sl+1;
 		/* .BR name (section)
